### PR TITLE
Fix 17201 - Update screen not updated after checking update

### PR DESCRIPTION
**Pull request by Fzzr**
This fix just involves re-running `_updateService.getInfo()` after an update check and download. It obviates my changes in 676c7ea5b3c61a296fba78da29a79728171d1a1d.

### CODE REVIEW

File: ui/inspectors/system-section.reel/updates/updates.reel/updates.js [hunk 1]
```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
 
     handleCheckDownloadAction: {
         value: function() {
-            this._updateService.checkAndDownload();
+            var self = this;
+            this._updateService.checkAndDownload().then(function() {
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/updates/updates.reel/updates.js]: As neither `checkAndDownload` or `getInfo` are used separately, they should probably be moved in a single `UpdateService` method.

File: ui/inspectors/system-section.reel/updates/updates.reel/updates.js [hunk 2]
```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
...
+                return self._updateService.getInfo();
+            }).then(function(info) {
+                self.info = info;
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/updates/updates.reel/updates.js:76]: `info` should be a `UpdateService` property that would be refreshed by the service without the inspector having to bother with that.

File: ui/inspectors/system-section.reel/updates/updates.reel/updates.js (hunk 1 above)
Comment (Fzzr) [reply ui/inspectors/system-section.reel/updates/updates.reel/updates.js]: `getInfo` is used separately in `enterDocument` for `updates.js`. I think it should stay that way.

Comment (pchaussalet) [reply ui/inspectors/system-section.reel/updates/updates.reel/updates.js]: right, but it should then be called directly by `checkAndDownload`

File: ui/inspectors/system-section.reel/updates/updates.reel/updates.js (hunk 2 above)
Comment (Fzzr) [reply ui/inspectors/system-section.reel/updates/updates.reel/updates.js:76]: For future reference: this turned out not to be useful, so instead we went back to the previous behavior of checkfetch also returning the new update data.

**Pull request merged**